### DISCUSSION TITLE
How to add your own images

### DISCUSSION THREAD

**Discussion by audreyhki** [USU-ITLS-x265-23-summer/class-repo#186] [Q&A] [answered]
Hello everybody, I hope your projects are going well! 

I have a quick question, I am having trouble adding a personal image as a background image on the site I am coding. The image is currently being hosted in Google Drive and that is where I got the URL from, 
 [https://drive.google.com/drive/folders/1vSMdooMSkWf4bVBw0HLXx8ljX1esD8Va](url)
When I insert this into my HTML code or CSS style sheet, I do not see the image on the Live Server. 
[https://drive.google.com/file/d/14CtWj8mMQxqmpwUiPawQECe8wiNfPwLW/view?usp=drive_link](url)

I have tried
`<img src="https://drive.google.com/file/d/14CtWj8mMQxqmpwUiPawQECe8wiNfPwLW/view?usp=drive_link">`
and
`body {
    background-image: url(https://drive.google.com/drive/folders/1vSMdooMSkWf4bVBw0HLXx8ljX1esD8Va); 
}

This is the image I am trying to use:
![5800DD5F-159F-4148-95A7-F7E90A0A2D21](https://github.com/USU-ITLS-x265-23-summer/class-repo/assets/133035062/2197e448-ea76-4949-aead-89fc75a02b70)
`

**Comment by jakegros** [comment, marked as answer]
Add the images into a folder like "assets" or "images" in your repo.  Then, use that as your source code. 

I don't know if this is the only way. It helped me, though, and seems a bit more streamlined.

**Reply by coopsand** [reply]
In my opinion, this way is really effective and it keeps things organized. I would suggest doing it like this way as well.

**Comment by jakegros** [comment]
I used this  `<img src="assets/Accessibility.svg.png">` since my image is save in my "assets" folder and then '/' followed by the name of image.  Let me know if this helps.

**Reply by audreyhki** [reply]
Thanks for the suggestion! This solved my problem and I was able to make significant progress on my project because of it!

**Reply by gabebott** [reply]
Thank you for sharing this! I had a similar issues!

**Comment by kassyoveson** [comment]
You can also try to make an image publicly accessible in Google Drive, by following these steps:

1- Open the image in Google Drive.
2- Click the Share button.
3- In the Share dialog box, click the Advanced link.
4- In the Advanced sharing settings dialog box, select the Anyone with the link can view option.
5- Click the Share button.

Once the image is publicly accessible, you can use the following code to add it as a background image on your site:
<body background="https://drive.google.com/file/d/14CtWj8mMQxqmpwUiPawQECe8wiNfPwLW/view?usp=drive_link">
</body>
This code will set the background image of the body element to the image you provided. The image will be displayed full-width and full-height, and it will repeat itself as needed.

If you want to control the size and position of the background image, you can use the CSS background-size and background-position properties. For example, the following code will set the background image to 50% of the width and height of the body element, and it will position the image at the top center of the body element using CSS:

body {
  background-image: url(https://drive.google.com/file/d/14CtWj8mMQxqmpwUiPawQECe8wiNfPwLW/view?usp=drive_link);
  background-size: 50%;
  background-position: top center;
}

I hope this helps.

**Reply by audreyhki** [reply]
Thanks for your suggestion Kassy! This was very clever of you and goes to show that there are multiple ways to solve a problem.

**Comment by beto0890** [comment]
Thank you for all the information that everyone provided, I was actually looking at how to do this.
